docs/CHANGELOG.asciidoc

```diff
@@ -90,10 +91,356 @@ For detailed information about the n.n.n release, review the following sections.
+=== Breaking change
+
+Breaking changes can prevent your application from optimal operation and performance. Before you upgrade to 7.12.0, review the breaking change, then mitigate the
+impact to your application.
+
+// The following section is re-used in the Installation and Upgrade Guide
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[maps-7.12.0-breaking-change]]
+.Removes the Geo threshold alert type
+[%collapsible]
+====
+*Details:* +
+Geo containment alerts and the Geo threshold alert type have similar functionality, so the Geo threshold alert type has been removed.
+
+*Impact* +
+Use Geo containment alerts.
+
+For more information, refer to {kibana-pull}89632[#89632].
+====
+
+// end::notable-breaking-changes[]
+
+[float]
+[[deprecation-v7.12.0]]
+=== Deprecations
+
+The following functionality is deprecated in 7.12.0, and will be removed in 8.0.0. Deprecated functionality does not have an immediate impact on your application, but 
+we strongly recommend you make the necessary updates after you upgrade to 7.12.0. 
+
+[discrete]
+[[alerting-7.12.0-deprecation]]
+.Deprecates `services.callCluster` in alerts and actions executors
+[%collapsible]
+====
+*Details* +
+Deprecates `services.callCluster` in the alert and action type executors. For more information, refer to {kibana-pull}86474[#86474].
+
+*Impact* +
+Use `services.scopedClusterClient`.
+====
+
+[discrete]
+[[management-7.12.0-deprecation]]
+.Deprecates configuring [kib] with a `server.host` value of `0`
+[%collapsible]
+====
+*Details* +
+In the new version of joi, `0` is an invalid hostname. When upgraded, the extra validate function can be removed. For more information, refer to {kibana-pull}87114[#87114].
+
+*Impact* +
+To bind to all interfaces, use `0.0.0.0`.
+====
+
+[discrete]
+[[platform-7.12.0-deprecation]]
+.Deprecates the use of `elasticsearch.logQueries`
+[%collapsible]
+====
+*Details* +
+When `logging.verbose: true` or `logging.root.level: 'debug'`, {es} query logs are now always shown. 
+In 7.11.0 and earlier, {es} query logs were not included in verbose logging unless `elasticsearch.logQueries` was configured. 
+For more information, refer to {kibana-pull}89296[#89296].
+
+*Impact* +
+To use verbose logging without {es} query logs, configure the query logs to `off` in kibana.yml:
+[source,js]
+----
+logging:
+  loggers:
+     - context: elasticsearch.query
+       level: off
+----
+====
+
+[float]
+[[feaatures-v7.12.0]]
+=== Features
+
+{kib} 7.12.0 adds the following new and notable features. For more information about the features introduced in 7.12.0, refer to 
+//{kibana-ref-all}/7.12/whats-new.html[What's new in 7.12].
+
+Alerting::
+* Adds Search alert {kibana-pull}88528[#88528]
+APM::
+* Correlations Beta (#86477) {kibana-pull}89952[#89952]
+Dashboard::
+* Adds the ability to copy a panel to another dashboard {kibana-pull}90856[#90856]
+Discover::
+* Adds "Hide chart" / "Show chart" persistence {kibana-pull}88603[#88603]
+{kib} Home & Add Data::
```

Comment:
This PR is more related to the app as a whole instead of Kibana Home and Add Data.

Also, this PR has a Release Note section in the PR.  How is that content being handled?

Reply:
This is more of a script question.